app/src/main/java/com/gh4a/adapter/timeline/CommentViewHolder.java

```diff
@@ -0,0 +1,197 @@
+    private final ReactionBar reactions;
+    private final PopupMenu mPopupMenu;
+    private final ReactionBar.AddReactionMenuHelper mReactionMenuHelper;
+
+    private TimelineItem.TimelineComment mBoundItem;
+
+    public interface Callback {
+        boolean canQuote();
+        void quoteText(CharSequence text);
+        boolean onMenItemClick(TimelineItem.TimelineComment comment, MenuItem menuItem);
+        List<Reaction> loadReactionDetailsInBackground(TimelineItem.TimelineComment item)
+                throws IOException;
+        Reaction addReactionInBackground(TimelineItem.TimelineComment item, String content)
+                throws IOException;
+    }
+
+    public CommentViewHolder(View view, HttpImageGetter imageGetter, String repoOwner,
+            ReactionBar.ReactionDetailsCache reactionDetailsCache, Callback callback) {
+        super(view);
+
+        mContext = view.getContext();
+        mImageGetter = imageGetter;
+        mCallback = callback;
+        mRepoOwner = repoOwner;
+
+        ivGravatar = (ImageView) view.findViewById(R.id.iv_gravatar);
+        tvDesc = (StyleableTextView) view.findViewById(R.id.tv_desc);
+        tvDesc.setMovementMethod(UiUtils.CHECKING_LINK_METHOD);
+        tvExtra = (StyleableTextView) view.findViewById(R.id.tv_extra);
+        tvTimestamp = (TextView) view.findViewById(R.id.tv_timestamp);
+        tvEditTimestamp = (TextView) view.findViewById(R.id.tv_edit_timestamp);
+        reactions = (ReactionBar) view.findViewById(R.id.reactions);
+        reactions.setCallback(this, this);
+        reactions.setDetailsCache(reactionDetailsCache);
+        ivMenu = (ImageView) view.findViewById(R.id.iv_menu);
+        ivMenu.setOnClickListener(this);
+
+        mPopupMenu = new PopupMenu(view.getContext(), ivMenu);
+        mPopupMenu.getMenuInflater().inflate(R.menu.comment_menu, mPopupMenu.getMenu());
+        mPopupMenu.setOnMenuItemClickListener(this);
+
+        MenuItem reactItem = mPopupMenu.getMenu().findItem(R.id.react);
+        mPopupMenu.getMenuInflater().inflate(R.menu.reaction_menu, reactItem.getSubMenu());
+
+        mReactionMenuHelper = new ReactionBar.AddReactionMenuHelper(view.getContext(),
+                reactItem.getSubMenu(), this, this, reactionDetailsCache);
+    }
+
+    @Override
+    public void bind(TimelineItem.TimelineComment item) {
+        mBoundItem = item;
+
+        User user = item.comment.getUser();
+        Date createdAt = item.comment.getCreatedAt();
+        Date updatedAt = item.comment.getUpdatedAt();
+
+        AvatarHandler.assignAvatar(ivGravatar, user);
+        ivGravatar.setTag(user);
+
+        tvTimestamp.setText(StringUtils.formatRelativeTime(mContext, createdAt, true));
+        if (createdAt.equals(updatedAt) || item.getCommitComment() != null) {
+            // Edit timestamp looks to be incorrect for commit comments. They all appear to be
+            // modified.
+            tvEditTimestamp.setVisibility(View.GONE);
```

Comment:
I'm not really sure why it's incorrect but almost all of the comments returned from pull request comments API call seem to have updated_at field set to the same time.

[Here](https://api.github.com/repos/Kotlin/KEEP/pulls/71/comments) most of them have it set to `2017-06-09T19:38:10Z` even though they do not appear with edited indicator on GitHub website.